client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -66,6 +75,24 @@ class ImageEditorToolbar extends Component {
 	onAspectOpen( event ) {
 		event.preventDefault();
 
+		const { imageMeetsMinimumDimensions, translate } = this.props;
+
+		if ( imageMeetsMinimumDimensions === false ) {
+			const noticeText = translate(
+				'To change aspect ratio, your image dimensions should be greater than %(width)px wide ' +
```

Comment:
This is throwing an error since sprintf needs a type for your args. eg `%(width)` vs `%(width)d`